packages/ui/src/button/index.css

```diff
@@ -1,45 +1,45 @@
-button {
+button.osc {
```

Comment:
Misschien bedoelde je hier `.osc button`? Nu moet je een `osc` class meegeven aan een button.

Reply:
Klopt, is voor alle components nu aangepast